node/modules/storageminer.go

```diff
@@ -724,6 +718,18 @@ func StorageAuth(ctx helpers.MetricsCtx, ca v0api.Common) (sectorstorage.Storage
 	return sectorstorage.StorageAuth(headers), nil
 }
 
+func StorageAuthWithURL(url string) func(ctx helpers.MetricsCtx, ca v0api.Common) (sectorstorage.StorageAuth, error) {
+	return func(ctx helpers.MetricsCtx, ca v0api.Common) (sectorstorage.StorageAuth, error) {
+		s := strings.Split(url, ":")
+		if len(s) != 2 {
+			return nil, errors.New("unexpected format of URL")
+		}
+		headers := http.Header{}
+		headers.Add("Authorization", "Bearer "+s[0])
+		return sectorstorage.StorageAuth(headers), nil
+	}
+}
```

Comment:
What kind of URL is this? It seems wrong to call it a URL, based on what we're doing here. Can you give an example value of what will be passed here?

Reply:
The string is `[jwtToken]:[url]`, the parameter should probably be called something like `apiInfo`